otter_welcome_buddy/common/constants.py

```diff
@@ -27,3 +35,5 @@ class CronExpressions(Enum):
 # Discord role that give access to the remaining channels and is
 # given when the user react to WELCOME_MESSAGES
 OTTER_ROLE: str = "Interviewee"
+# URL to the header image used for branding or visual identity in the Discord server
+OTTER_HEADER_IMAGE: str = "https://shorturl.at/blqMV"
```

Comment:
Why do you need the image that says "welcome to the discord"? I still suggest to use the image from my previous comment.